src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -24,7 +24,7 @@ internal sealed partial class LanguageParser : SyntaxParser
         // can be reused (hence pooled) since the syntax factory methods don't keep references to
         // them
 
-        private readonly SyntaxListPool _pool = new SyntaxListPool(); // Don't need to reset this.
+        private readonly SyntaxListPool _pool = SyntaxListPool.GetInstance(); // Don't need to reset this.
```

Comment:
Trying to understand if the comment here is still valid. There is no resetting in the pool and it's a ctor call all the time. But the comment had me digging into the pool to see if we now needed to reset state since there was a pool and not always a `new`. 

@cston, @CyrusNajmabadi